src/fsharp/ast.fs

```diff
@@ -1742,7 +1755,7 @@ let mkSynDotBrackSeqSliceGet  m mDot arr (argslist:list<SynIndexerArg>) =
                        | SynIndexerArg.One x -> yield x
                        | _ -> () ]
     if notsliced.Length = argslist.Length then
-        SynExpr.DotIndexedGet(arr,[SynIndexerArg.One (SynExpr.Tuple(notsliced,[],unionRanges (Seq.head notsliced).Range (Seq.last notsliced).Range))],mDot,m)
```

Comment:
I agree we should make a proto fsharp.core build.  I will add an issue, it is beyond the scope of this checkin, which is already 10 times bigger than I expected :-)